### PR TITLE
ui: Updated units names

### PR DESCRIPTION
**Pull request by OlegChuev**
### Description

This PR...

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [X] Trivial

#### How did you try to break this feature and the system with this change?

Before the fix
image

After the fix
image

### CODE REVIEW

File: ui/public/locales/en.json [hunk 1]
```diff
@@ -1197,7 +1197,7 @@
 "label.maxinstance": "Max instances",
 "label.maxiops": "Max IOPS",
 "label.maxmembers": "Max members",
-"label.maxmemory": "Max. memory (MiB)",
+"label.maxmemory": "Max. memory (MB)",
```
Comment (DaanHoogland) [inline ui/public/locales/en.json:1200]: are we sure it is not in multitudes of 1024*1024 instead of 1.000.000?

Comment (rohityadavcloud) [reply ui/public/locales/en.json:1200]: MiB is appropriate here; our memory calculations are not dividing bytes by 10^6. Is there a specific place you're seeing something not expected?

**Comment by DaanHoogland** [comment]
@OlegChuev any opinion on our comments? we are planning for 4.18.2 and it would be nice to resolve this before then.

File: ui/public/locales/en.json (hunk 1 above)
Comment (rohityadavcloud) [reply ui/public/locales/en.json:1200]:
```suggestion
"label.maxmemory": "Max. memory (MiB)",
```

File: ui/public/locales/en.json [hunk 2]
```diff
@@ -1253,7 +1253,7 @@
 "label.miniops": "Min IOPS",
 "label.minmaxiops": "Min IOPS / Max IOPS",
 "label.minmembers": "Min members",
-"label.minmemory": "Min memory (in MB)",
+"label.minmemory": "Min. memory (MB)",
```
Comment (rohityadavcloud) [inline ui/public/locales/en.json:1256]:
```suggestion
"label.minmemory": "Min. memory (MiB)",
```

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan ui

**Review by shwstppr — APPROVED** [review]

**Pull request merged**